client/src/test/suite/extension.test.ts

```diff
@@ -174,6 +174,7 @@ suite('Extension Test Suite', function (this: Suite) {
   })
   test('show cnxml preview with no file open', async () => {
     assert.strictEqual(vscode.window.activeTextEditor, undefined)
+    await sleep(500) // FIXME: Make me go away (see https://github.com/openstax/cnx/issues/1569)
```

Comment:
I added this only because it seemed like client tests (`npm run test:client`) were consistently failing due to a race condition.